addon/components/au-accordion.hbs

```diff
@@ -4,24 +4,24 @@
       <div>
         <AuButton @skin="tertiary" aria-hidden="true" focusable="false" aria-expanded="{{if this.openAccordion "true" "false"}}">
           {{#if @hideText}}
-            <span class="au-u-hidden-visually">{{@accordionButtonLabel}}</span>
+            <span class="au-u-hidden-visually">{{@label}}</span>
           {{else}}
-            {{@accordionButtonLabel}}
+            {{@label}}
           {{/if}}
         </AuButton>
         <p>
-          {{@accordionSubTitle}}
+          {{@subTitle}}
```

Comment:
I would go with `subtitle` since it's one word in English :smile:.

Reply:
ah I was really thinking about that! but saw that it was already camel case so was like yeah it should probably be like that I guess? I agree, will change it right away